### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds a "before packaging day" section, with some new checklist items.
Makes this one Uyuni specific. SUMA PR in spacewalk repo: https://github.com/SUSE/spacewalk/pull/12351. Renames file.

### CODE REVIEW

**Comment by Loquacity** [comment]
Also adds a few more fullstops, just for @keichwa :wink:

**Comment by Loquacity** [comment]
@jcayouette @keichwa can I get some eyeballs on this please?

**Review by jcayouette — CHANGES_REQUESTED** [review]
I adjusted the release day notes for publishing to gh-pages. Feel free to clean up as you like. :)

File: .github/release_issue_template.md [hunk 1]
```diff
@@ -24,5 +24,6 @@ Check with the docs squad coordinator for confirmation.
 
 - [ ] Build both SUMA and Uyuni docs from release branch and visually check output.
 - [ ] Publish to documentation.suse.com: https://github.com/uyuni-project/uyuni-docs/wiki/publishing-to-enterprise-endpoints
-- [ ] Publish to opensource.suse.com: https://github.com/uyuni-project/uyuni-docs/wiki/publishing-to-master-branches-on-github-pages
+- [ ] For Uyuni only - Build latest Uyuni docs with `make antora-uyuni`.
+- [ ] Create a new uyuni-YYYY-MM-pages branch based on the gh-pages branch(https://github.com/uyuni-project/uyuni-docs/tree/gh-pages). Add Julio as the Assignee if available so he can publish when ready.(Use a label with current Uyuni version (uyuni-2020.09)
```
Comment (Loquacity) [inline .github/release_issue_template.md:28]: Do we need to mention that we only put the build dir in that branch? that had me scratching my head when i did it ;)

Comment (jcayouette) [reply .github/release_issue_template.md:28]: Ah you are right. We should mentioned that we place the static content there. Also we need to make sure that **.nojekyll** is in the dir.

**Review by jcayouette — APPROVED** [review]

**Pull request merged**